sdk/storage/storage-file-datalake/src/generated/src/storageClientContext.ts

```diff
@@ -11,7 +11,7 @@
 import * as coreHttp from "@azure/core-http";
 
 const packageName = "azure-storage-datalake";
-const packageVersion = "1.0.0";
+const packageVersion = "12.4.0";
```

Comment:
Don't change this one. We don't pass the version when `build:autorest` so this version will always be "1.0.0" and it's actually better.